src/components/Views/ObservationEditor/ObservationEditor.js

```diff
@@ -496,6 +516,76 @@ class ObservationEditor extends React.Component<
     }
   };
 
+  observationExists = () => {
```

Comment:
bleh, since we're doing this might be easier to just pass in the observations as is and then turn it into an array in the component. This seems expensive